server.js

```diff
@@ -125,7 +125,7 @@ ${beautify(dcrResp.data, null, 4)}`
 		};
 
 		let accessTokenResp = await axios.post(
-			`https://${conf.km_hostname}:${conf.km_port}/oauth2/token`,
+			`https://${conf.km_hostname}:${conf.tokenPort}/token`,
```

Comment:
Let's revert this change back and keep it pointing to the `/oauth2/token` endpoint of the Key Manager node. This is made because to communicate with the Key Manager node directly in a distributed environment.